controllers/chat/group/changeChannelDetail.js

```diff
@@ -62,6 +63,7 @@ const changeChannelDetail = async (req, res) => {
         text: textOtherUser,
         own_text: textOwnUser,
         other_text: textTargetUser,
+        better_type: 'change_channel_detail',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the addition of the `better_type` property here also needs clarification. If it's meant to replace the existing `type` property, then changing the value of `type` would be a better approach.

```diff
-        better_type: 'change_channel_detail',
+        type: 'change_channel_detail',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->